content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -64,6 +68,58 @@
     margin-top: -($feed-gutter-v / 2 + 18 / 2);
     cursor: pointer;
   }
+
+  .action-items-container {
+    background: $white;
```

Comment:
The spec gives us `#FBFCFC` for this although i think `$bg-grey` is close enough